packages/components/src/navigation/NavTabList.scss

```diff
@@ -0,0 +1,166 @@
+@import '../../scss/custom.scss';
+
+$tab-height: 32px;
+$tab-drag-border-width: 1px;
+$tab-font-size: 1rem;
+
+$tab-link-side-padding: 24px;
+$tab-link-underline-spacing: 6px;
+
+$tab-link-hover-underline-color: $gray-400;
+
+$tab-link-active-color: $gray-200;
+$tab-link-active-underline-color: var(--dh-color-accent-bg);
+
+$tab-link-active-hover-underline-color: var(--dh-color-accent-bg);
+
+$tab-button-hover-color: $gray-200;
+$tab-button-separator-color: $gray-600;
+
+$tab-control-btn-offset: -8px;
+
+.nav-container {
+  display: flex;
+  flex-shrink: 0;
+
+  .nav-tabs {
+    --dragging-ant-color: $gray-300;
```

Comment:
unused?

Reply:
Ya I think I initially tried to pass this to the `ants-base` mixin, but it can't handle css variables properly